build_examples/build_cpu_mpi-only_gcc_ubuntu20.04.sh

```diff
@@ -41,7 +41,7 @@ HDF5_LIB_FLAGS="-lhdf5_serial_fortran -lhdf5_serialhl_fortran -lhdf5_serial -lhd
 # Please set the compile flags based on your compiler and hardware setup.
 ###########################################################################
 
-FFLAGS="-O3 -march=native"
+FFLAGS="-O3 -march=native $(mpif90 --showme:compile) $(mpif90 --showme:link)"
```

Comment:
It will be nice to keep the original `FFLAGS` and comment it out instead of deleting it? Same thing for the build script [build_examples/build_cpu_mpi+multithread_gcc_ubuntu20.04.sh](https://github.com/gxyd/POT3D/pull/24/files#diff-2352c8884be1fe635673a3e6feed021edc05c72309d263a938ec2cf25a089ac5)

That will be make it easy for everyone to see the original setting of `FFLAGS` in POT3D repository.